Sources/PaymentsCore/Networking/Protocols/APIRequest.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+public protocol APIRequest {
```

Comment:
From reading through the code, I wouldn't think that `APIRequest` was a protocol but rather a class. I was thinking .. how can we name this to be more "protocolly". Here's what[ Swift's guidelines ](https://swift.org/documentation/api-design-guidelines/)say:

```
Protocols that describe what something is should read as nouns (e.g. Collection).

Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).
```

I guess this falls in the first category and that technically the name fits the description above. However, just posting this in case anyone else also had this confusion.

Reply:
naming is hard! 😆 open to alternatives, but it feels okay to me - it feels like it defines the contract for what constitutes an API request well